app.js

```diff
@@ -66,6 +67,8 @@ app.get('/view', async (req, res) => {
     let artistResult = await getTopArtists(spotifyApi, TIME_RANGE);
     let genresResult = await getTopGenres(spotifyApi, TIME_RANGE);
     let currentUser = await getCurrentUser(spotifyApi, TIME_RANGE);
+    let topDecade = await getMostListenedToDecade(spotifyApi);
+    let decadeListKeys = Object.keys(topDecade)
```

Comment:
could you set decadeListKeys: Object.keys(topDecade) in the actual index.ejs? I'll comment there too